Make travis pass

https://travis-ci.com/SymbiFlow/python-sdf-timing/jobs/190714510
```
0.01s$ source ~/virtualenv/python2.7/bin/activate
$ python --version
Python 2.7.14
$ pip --version
pip 9.0.1 from /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (python 2.7)
install
2.42s$ pip install tox
Collecting tox
  Downloading https://files.pythonhosted.org/packages/d2/68/d8a9466c8ecda020b36cd5f5d0c274403941a401fe613d868ecf5fc84dd5/tox-3.8.6-py2.py3-none-any.whl (73kB)
    100% |████████████████████████████████| 81kB 2.9MB/s 
Requirement already satisfied: py<2,>=1.4.17 in /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (from tox)
Requirement already satisfied: six<2,>=1.0.0 in /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (from tox)
Requirement already satisfied: setuptools>=30.0.0 in /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (from tox)
Collecting filelock<4,>=3.0.0 (from tox)
  Downloading https://files.pythonhosted.org/packages/2a/bd/6a87635dba4906ae56377b22f64805b2f00d8cafb26e411caaf3559a5475/filelock-3.0.10.tar.gz
Collecting toml>=0.9.4 (from tox)
  Downloading https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting virtualenv>=1.11.2 (from tox)
  Downloading https://files.pythonhosted.org/packages/33/5d/314c760d4204f64e4a968275182b7751bd5c3249094757b39ba987dcfb5a/virtualenv-16.4.3-py2.py3-none-any.whl (2.0MB)
    100% |████████████████████████████████| 2.0MB 665kB/s 
Requirement already satisfied: pluggy<1,>=0.3.0 in /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (from tox)
Building wheels for collected packages: filelock
  Running setup.py bdist_wheel for filelock ... done
  Stored in directory: /home/travis/.cache/pip/wheels/46/b3/26/8803692ec1f1729fcf201583b5de74f112da1f1488f36e47b0
Successfully built filelock
Installing collected packages: filelock, toml, virtualenv, tox
Successfully installed filelock-3.0.10 toml-0.10.0 tox-3.8.6 virtualenv-16.4.3
10.50s$ tox
GLOB sdist-make: /home/travis/build/SymbiFlow/python-sdf-timing/setup.py
py27 create: /home/travis/build/SymbiFlow/python-sdf-timing/.tox/py27
py27 installdeps: check-manifest, pyjson, flake8, pytest
py27 inst: /home/travis/build/SymbiFlow/python-sdf-timing/.tox/.tmp/package/1/sdf_timing-0.0.1.zip
py27 installed: DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.,atomicwrites==1.3.0,attrs==19.1.0,check-manifest==0.37,configparser==3.7.4,entrypoints==0.3,enum34==1.1.6,flake8==3.7.7,funcsigs==1.0.2,functools32==3.2.3.post2,mccabe==0.6.1,more-itertools==5.0.0,pathlib2==2.3.3,pluggy==0.9.0,ply==3.11,py==1.8.0,pycodestyle==2.5.0,pyflakes==2.1.1,pyjson==1.3.0,pytest==4.4.0,scandir==1.10.0,sdf-timing==0.0.1,six==1.12.0,typing==3.6.6
py27 run-test-pre: PYTHONHASHSEED='3997939947'
py27 run-test: commands[0] | check-manifest --ignore 'tox.ini,tests*'
lists of files in version control and sdist match
py27 run-test: commands[1] | python setup.py check -m -s
running check
py27 run-test: commands[2] | flake8 --ignore F401 .
./sdf_timing/sdfparse.py:8:1: E302 expected 2 blank lines, found 1
./sdf_timing/sdfparse.py:9:5: F821 undefined name 'sdflex'
./sdf_timing/sdfparse.py:10:26: F821 undefined name 'sdflex'
./sdf_timing/sdfparse.py:13:1: E302 expected 2 blank lines, found 1
./sdf_timing/sdflex.py:74:1: E302 expected 2 blank lines, found 1
./sdf_timing/sdflex.py:78:1: E305 expected 2 blank lines after class or function definition, found 1
ERROR: InvocationError for command /home/travis/build/SymbiFlow/python-sdf-timing/.tox/py27/bin/flake8 --ignore F401 . (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py27: commands failed
The command "tox" exited with 1.
Done. Your build exited with 1.
``

--- a/sdf_timing/sdflex.py
+++ b/sdf_timing/sdflex.py
@@ -71,8 +71,10 @@ def t_error(t):
 # input is the input text string
 # token is a token instance
 
+
 def find_column(input, token):
     line_start = input.rfind('\n', 0, token.lexpos) + 1
     return (token.lexpos - line_start) + 1
+
 
 lexer = lex.lex()

--- a/sdf_timing/sdfparse.py
+++ b/sdf_timing/sdfparse.py
@@ -2,13 +2,16 @@
 
 import argparse
 import json
+import sdflex
 
 import sdfyacc
+
 
 def parse(input):
     sdflex.input_data = input
     sdfyacc.parser.parse(sdflex.input_data)
     return sdfyacc.timings
+
 
 def main():
     parser = argparse.ArgumentParser()

